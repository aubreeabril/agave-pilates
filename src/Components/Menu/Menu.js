import React from 'react'
import Container from 'react-bootstrap/Container'
import Col from 'react-bootstrap/Col'
import Image from 'react-bootstrap/Image'
import Row from 'react-bootstrap/Row'
import logo from '../Menu/png_large_textAsset 5xhdpi.png'
import '../../App.css'



const Menu = (props) => (
    <React.Fragment>
        <Container className='d-md-none'>
            <Row>
                <Col md={12} className='collapsed-menu'>
                    <Image src={logo} alt='Illustration of an agave plant' style={{ maxWidth: '30%', paddingBottom: '10px'}} />
                </Col>
            </Row>
            <Row>
                <Col md={12} className='collapsed-menu'>
                    <h4>About Jenny</h4>  
                </Col>           
            </Row>
            <Row>
                <Col md={12} className='collapsed-menu'>
                    <h4>Scheduling</h4>
                </Col>
            </Row>
        </Container>
        <Container className='d-none d-md-block' style={{ paddingBottom: '5%'}}>
            <Row>
                <Col>
                    <h2 className='uncollapsed-menu-text'>About Jenny</h2>
                </Col>
                <Col>
                    <Image src={logo} alt='Illustration of an agave plant' style={{ maxWidth: '55%', paddingTop: '10%'}} />
                </Col>
                <Col>
                    <h2 className='uncollapsed-menu-text'>Scheduling</h2>
                </Col>
            </Row>
        </Container>
    </React.Fragment>
)

export default Menu