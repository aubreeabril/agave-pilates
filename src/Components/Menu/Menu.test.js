import React from 'react'
import { render } from '@testing-library/react'
import Menu from './Menu'

describe('<Menu />', () => {
    it('renders the component', () => {
        const menu = render(<Menu />)
        expect(menu.baseElement).toMatchSnapshot()
    })
})