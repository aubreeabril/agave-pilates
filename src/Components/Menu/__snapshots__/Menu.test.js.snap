// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Menu /> renders the component 1`] = `
<body>
  <div>
    <div
      class="d-md-none container"
    >
      <div
        class="row"
      >
        <div
          class="collapsed-menu col-md-12"
        >
          <img
            alt="Illustration of an agave plant"
            class=""
            src="png_large_textAsset 5xhdpi.png"
            style="max-width: 30%; padding-bottom: 10px;"
          />
        </div>
      </div>
      <div
        class="row"
      >
        <div
          class="collapsed-menu col-md-12"
        >
          <h4>
            About Jenny
          </h4>
        </div>
      </div>
      <div
        class="row"
      >
        <div
          class="collapsed-menu col-md-12"
        >
          <h4>
            Scheduling
          </h4>
        </div>
      </div>
    </div>
    <div
      class="d-none d-md-block container"
      style="padding-bottom: 5%;"
    >
      <div
        class="row"
      >
        <div
          class="col"
        >
          <h2
            class="uncollapsed-menu-text"
          >
            About Jenny
          </h2>
        </div>
        <div
          class="col"
        >
          <img
            alt="Illustration of an agave plant"
            class=""
            src="png_large_textAsset 5xhdpi.png"
            style="max-width: 55%; padding-top: 10%;"
          />
        </div>
        <div
          class="col"
        >
          <h2
            class="uncollapsed-menu-text"
          >
            Scheduling
          </h2>
        </div>
      </div>
    </div>
  </div>
</body>
`;
